app/test/frontend/backend_test.dart

```diff
@@ -1049,3 +1049,16 @@ void main() {
     }, timeout: Timeout.factor(2));
   });
 }
+
+class TestDelayCompletion {
```

Comment:
Could we give this some docs, I can't immediately see what it does :)

Reply:
I didn't see it either, but I hope that moving tests to fake_gcloud would somehow eliminate the need for it :)